shared/js/content-scripts/content-script.js

```diff
@@ -49,15 +49,15 @@ function getLocation () {
     // content script only works from window.top and will throw
     // errors if it runs in iframes. Try to access hostname and
     // bail if there are any errors.
+    if (mainFrameURL) { return mainFrameURL }
+
     try {
-        window.top.location.hostname
+        var loc = window.top.location
+        mainFrameURL = loc.protocol + '//' + loc.hostname + loc.pathname
```

Comment:
Lint was complaining about having a bare var reference without any operations inside the `try` statement. I decided it's better to just move the bits we need in the `try`.